knack/iframeMapMessenger.js

```diff
@@ -79,9 +72,12 @@
       if (data.message === "LAT_LON_FIELDS") {
         console.log("received message ", data);
         var $latLonFields = $("#kn-input-field_3300");
-
-        $latLonFields.find("#latitude").val(data.lat);
-        $latLonFields.find("[name='longitude']").val(data.lng);
+        if (!!data.lat && !!data.lng) {
+          $latLonFields.find("#latitude").val(data.lat);
+          $latLonFields.find("[name='longitude']").val(data.lng);
```

Comment:
i am wondering why you need a difference accessor for lat and lon, but am also too lazy to find out for myself 🙈

Reply:
Never thought to investigate why it was written like this, but now my interest is piqued so 
![Screenshot 2025-06-27 at 10 15 42 AM](https://github.com/user-attachments/assets/6b188050-b683-4c12-a3b0-16d4f3ed2ec3)

Reply:
😵